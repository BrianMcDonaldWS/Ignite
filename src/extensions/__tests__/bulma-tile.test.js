import MarkdownIt from 'markdown-it';
import tile from '../bulma-tile';

const md = new MarkdownIt().use(tile);

test('bulma - tile', () => {
  expect(
    md.render(`
::: tile

# Title

## I can contain various combinations of elements.
:::
  `)
  ).toMatchSnapshot();

  expect(
    md.render(`
::: tile is-warning notification

# Title

## I can contain various combinations of elements.
:::
  `)
  ).toMatchSnapshot();
});
