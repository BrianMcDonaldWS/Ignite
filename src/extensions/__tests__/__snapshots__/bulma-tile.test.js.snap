// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bulma - tile 1`] = `
"
        <div class=\\"tile \\">
      <h1>Title</h1>
<h2>I can contain various combinations of elements.</h2>

      </div>
    "
`;

exports[`bulma - tile 2`] = `
"
        <div class=\\"tile is-warning notification\\">
      <h1>Title</h1>
<h2>I can contain various combinations of elements.</h2>

      </div>
    "
`;
